deploy/ansible/provision.yml

```diff
@@ -0,0 +1,134 @@
+- hosts: all
+  become: yes
+  become_method: sudo
+  gather_facts: no
+  vars_files:
+    - vars/main.yml
+    - "vars/{{env}}.yml"
+  tasks:
+    # Start managing system packages and updates.
+    - name: ensure python is installed
```

Comment:
Ubuntu 16 doesn't ship with python installed, which is a prereq for Ansible. We'll always need it for now!